crates/core/src/ring/connection_manager.rs

```diff
@@ -2,26 +2,28 @@ use dashmap::DashMap;
 use parking_lot::Mutex;
 use rand::prelude::IndexedRandom;
 use std::collections::{btree_map::Entry, BTreeMap};
+use std::net::SocketAddr;
 use std::sync::atomic::{AtomicUsize, Ordering};
+use std::time::{Duration, Instant};
 
 use crate::topology::{Limits, TopologyManager};
 
 use super::*;
-use std::time::{Duration, Instant};
 
 #[derive(Clone)]
 pub(crate) struct TransientEntry {
+    /// Entry tracking a transient connection that hasn't been added to the ring topology yet.
+    /// Transient connections are typically unsolicited inbound connections to gateways.
     #[allow(dead_code)]
```

Comment:
why this is dead_code even, we should remove it, and if not needed remove the field

Reply:
Removed the unused `opened_at` field and the dead_code allow on `is_gateway`; transient TTL is enforced in the bridge task, so we don’t need to store the open timestamp here.